src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/useBrevmottakerSkjema.ts

```diff
@@ -185,6 +197,13 @@ export const useBrevmottakerSkjema = ({ eksisterendeMottakere }: Props) => {
         settNavnErPreutfylt(skalNavnVærePreutfylt);
     }, [mottaker.verdi, land.verdi]);
 
+    useEffect(() => {
+        if (land.verdi !== 'NO') {
+            postnummer.nullstill();
+            poststed.nullstill();
+        }
+    }, [land.verdi, postnummer.verdi, poststed.verdi]);
```

Comment:
Hadde det vært bedre å ikke bruke en useEffect her? Og trenger vi egentlig å lytte på noe annet enn når land.verdi endrer seg?
Se eksempel fra denne artikkelen: https://react.dev/learn/you-might-not-need-an-effect
![image](https://github.com/user-attachments/assets/472da731-d7b5-44fa-b8e7-2fb3ddbffacf)

Reply:
Helt enig! Fjernet `useEffect` nå 😊